web/assets/js/views/PhysicalEvaluation-0.3-peds.js

```diff
@@ -861,6 +863,36 @@ let viewExtension = Backbone.View.extend({
         }
         return result;
     },
+    handleProtocolModificationAllCheck: function (e) {
+        let block = $(e.currentTarget).closest(".modification-block");
+        let primarySelect = block.find(".modification-select select");
+        let modificationType = $(e.currentTarget).parents(".modification-select").data("modification-type");
+        if ($(e.currentTarget).is(":checked")) {
+            let elements = $(modificationType + "-select")
+                .find("select")
+                .not(primarySelect)
+                .val(primarySelect.val())
+                .closest(".modification-block");
+            for (let i = 0; i < elements.length; i++) {
+                this.showModificationBlock($(elements[i]));
+                this.handleProtocolModificationBlock($(elements[i]));
+                this.triggerEqualize();
+                this.displayWarning($(elements[i]));
+            }
+        } else {
+            let elements = $(modificationType + "-select")
+                .find("select")
+                .not(primarySelect)
+                .val("")
+                .closest(".modification-block");
+            for (let i = 0; i < elements.length; i++) {
```

Comment:
Since this for loop is same as above, could you extract this out of the if condition?